apps/src/sites/studio/pages/code-studio.js

```diff
@@ -17,6 +17,9 @@ import hashEmail from '@cdo/apps/code-studio/hashEmail';
 import GDPRDialog from '@cdo/apps/templates/GDPRDialog';
 import getScriptData from '@cdo/apps/util/getScriptData';
 
+// Conditionally enable "why-did-you-render" dev experiment
+import '@cdo/apps/util/whyDidYouRender';
```

Comment:
Generally speaking, adding things to this file is an anti-pattern, because webpack wants dependencies to be imported near the place they are needed rather than in the global scope. However, given that what you are trying to do here is global in nature, this seems much more excusable than most of the other imports in this file.